lib/generators/templates/landable.rb

```diff
@@ -59,8 +59,21 @@
   # Also specify which queue to publish the events to in RabbitMq. By default
   # event are published to the event queue
 
-  # config.enable_hutch = true
-  # config.event_queue = 'events'
+  # enable hutch
+  #   config.hutch_enable = true
```

Comment:
I think this should remain `enable_hutch` because it denotes whether or not we are going to enable the hutch service with Landable.